js-intro.js

```diff
@@ -55,7 +106,19 @@ var notAString2 = 42
 
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
-
+var onlyCats = toonimals.filter(value => value.animal === "cat")
+console.log(onlyCats)
```

Comment:
This is a variable which gives us the correct result, but we need this logic to be wrapped in a function in order for it to be reusable